.pre-commit-config.yaml

```diff
@@ -22,6 +22,6 @@ repos:
     hooks:
       - id: isort
   - repo: https://github.com/psf/black
-    rev: 21.9b0
+    rev: 22.3.0
```

Comment:
Not sure about bumping this here. @sanderegg @pcrespov ?

Reply:
The pre-commit scripts run on their own virtualenv (with is different from our ``.venv``) so bumping the version there is ok. Nonetheless there should be some level of compatibility with the tools used in vscode (which are installed in our ``.venv``).

This number should be at least compatible (if possible identical) with the version we have in the ``requirements/_tool.txt`` (see screenshot below)... otherwise it is very confusing to format in one way with vscode tooling and then again with the pre-commit.  Same applies for ``isort``. 

That said, as you can see we use ``22.1.0`` so that hsould be the version you put there.

![image](https://user-images.githubusercontent.com/32402063/161989144-fb7fd505-1834-4109-9d0c-810154b7c0b4.png)